drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -675,6 +675,32 @@ static int stm32_clock_control_init(const struct device *dev)
 	/* Update CMSIS variable */
 	SystemCoreClock = CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC;
 
+#if defined(CONFIG_CLOCK_STM32_PLL2_ENABLE)
+	/* Configure PLL2 */
+	/* Select clock source */
+	/* Init pre divider DIVM */
+	LL_RCC_PLL2_SetM(CONFIG_CLOCK_STM32_PLL2_M_DIVISOR);
+
+	/* Config PLL */
+	/* VCO sel, VCO range */
+	LL_RCC_PLL2_SetVCOInputRange(LL_RCC_PLLINPUTRANGE_1_2);
+	LL_RCC_PLL2_SetVCOOutputRange(LL_RCC_PLLVCORANGE_WIDE);
```

Comment:
Input/Output range should be calculated dynamically

Reply:
Thanks, I did it like you did for PLL3